specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -2123,6 +2124,24 @@
           "description": "Percentage completed.",
           "readOnly": true
         },
+        "value": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
```

Comment:
LAC results actually wouldn't have an id. Might want to just not define a schema for the elements of the items array since it can be fluid.